_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,445 @@
+- it is easy and efficient to add features to OSS, which can be freely shared with anyone, for any purpose. This means that everyone can benefit.
+
+The process for the development of open standards, similar to OSS, is open and transparent to all interested parties.
+It cannot be controlled by any single person or entity with any vested interests, because:
+
+- open standards are platform independent, vendor neutral and usable for multiple implementations;
+- specifications and supporting material are freely available with limited restrictions; and
+- they are supported by the community and demonstrate independence or approved through due process by rough consensus among participants.
+
+Working across the ESDC, and joined by other GC departments, provincial Governments, companies and the public at large, we can build new, innovative digital solutions that support our common policies and activities, and work towards technological sovereignty.
+
+Open standards and OSS allows for incremental innovation, based on the sharing of knowledge and skills.
+Openness increases trust in public services.
+It helps to solve complex technological problems by getting others to contribute unexpected solutions.
+
+Being free to inspect and improve, OSS offers more options for increasing security.
+It allows for independent audits and code inspections, so the time and effort spent can be balanced according to need.
+This improves security, not just for us, but for everyone.
+
+This "Open First" strategy reinforces the ESDC's internal working culture that is already largely based on the principles of working in the open, open standards ans OSS.
+However, it will also help ESDC change some of its technological and information management processes.
+
+To build Canada together, we have to get ready to contribute, and be ready to accept contributions from citizens.
+
+The strategy is a practical instrument for achieving digital transformation in IITB, ESDC and the GC.
+At the same time, in a world of increasing uncertainty, complexity and change, the co-creation of technology increases our capacity to act independently to safeguard our interests, uphold Canadian values and way of life, and help shape our future.
+
+This strategy sets out how we encourage and leverage open principles and development practices to co-create, share and reuse across ESDC and the GC.
+By removing legal barriers, we can make it easier to contribute to OSS projects and to share our own solutions.
+The strategy brings ESDC closer to the OSS community, and makes us an agile contributor and an active participant.
+It encourages us to explore opportunities for dedicated support services for open solutions that we consider critical.
+We will make more and more use of open standards and OSS – in areas where it is practical, in the domain of IT, to areas where it makes sense and domains where it is strategic.
+
```

Comment:
It was the Canadian Government's involvement in the fields of aviation, telephony, and nuclear technologies that spurred innovation and new industries in our country.  The Government is once again well positioned to rally the country into the growing IT field of Open Source that employs tens of thousand of people across the country.  As a strong supporter of the GC's vision of Open source, ESDC can play a role in bringing this vision to fruition through the development and implementation of this strategy.